src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2404,14 +2410,24 @@ private static void AppendLoadDirectiveDiagnostics(DiagnosticBag builder, Syntax
 
         // Do the steps in compilation to get the method body diagnostics, but don't actually generate
         // IL or emit an assembly.
-        private void GetDiagnosticsForAllMethodBodies(DiagnosticBag diagnostics, CancellationToken cancellationToken)
+        private void GetDiagnosticsForAllMethodBodies(DiagnosticBag diagnostics, bool doLowering, CancellationToken cancellationToken)
```

Comment:
>doLowering [](start = 86, length = 10)

nit: This parameter probably could use a comment

Reply:
> nit: This parameter probably could use a comment

Please provide suggested wording beyond what the name already implies.

---
In reply to: [350927999](https://github.com/dotnet/roslyn/pull/40010#discussion_r350927999) [](ancestors = 350927999)